openquakeplatform/static/geonode/css/d3.css

```diff
@@ -98,4 +98,8 @@ svg {
 }
 .grid path {
   stroke-width: 0;
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> origin
```

Comment:
Conflict must be fixed.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/d26a8e6dda9af9e90ba7eedd4c5e9b09e2964ad2